Server-Protocol-Buffer/build.gradle

```diff
@@ -5,9 +5,6 @@ sourceSets {
     main.java.srcDirs += "${protobuf.generatedFilesBaseDir}/main/javalite"
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
I added this while working on screenshots , please double check? run the screenshots task if necessary

Reply:
Since adding Coil, it would not work like this.

I found this via:
https://stackoverflow.com/questions/31870426/how-to-set-source-version-of-java-library-in-android-studio-project

which seems to indicate that there is a bug when using pure java libraries as submodule in Android projects.

I'll check the screenshots again.

Reply:
Screenshots fail, but because of something else:
https://app.circleci.com/pipelines/github/corona-warn-app/cwa-app-android/18215/workflows/b290dbf6-db4d-47cd-9341-3e844f078091/jobs/155003

Reply:
https://github.com/corona-warn-app/cwa-app-android/pull/3760/commits/cdc99618768a26b8795e4c12c50edb94f755b454

Should fix screenshots: Missing mocks and qrcode generation.

`FakeImageLoader.kt` offers methods to get the same image loading as in the live app, or override to always get the same. I added two examples, one for qrcodes and one for `R.drawable.*`

Reply:
New  screenshot test is running:
https://app.circleci.com/pipelines/github/corona-warn-app/cwa-app-android/18235/workflows/97ed8c1c-a451-43c3-b4da-d8744ebeca0b/jobs/155229